ui/lib/hooks.js

```diff
@@ -54,19 +53,16 @@ export function useUser({ redirectTo, redirectIfFound } = {}) {
       }
 
       // Clear user cache
-      mutate(undefined, false)
+      mutate(undefined, true)
```

Comment:
This should have been set to `true` since it will force revalidation: https://github.com/vercel/swr/blob/3f904887eda14861179d70afe7ad9bbdf5c00863/_internal/utils/mutate.ts#L44